airbyte-cdk/python/airbyte_cdk/connector_builder/message_grouper.py

```diff
@@ -230,17 +235,17 @@ def _read_stream(self, source: DeclarativeSource, config: Mapping[str, Any], con
             yield AirbyteTracedException.from_exception(e, message=error_message).as_airbyte_message()
 
     @staticmethod
-    def _parse_json(log_message: AirbyteLogMessage):
+    def _parse_json(log_message: AirbyteLogMessage) -> Optional[Dict[str, Any]]:
         # TODO: As a temporary stopgap, the CDK emits request/response data as a log message string. Ideally this should come in the
         # form of a custom message object defined in the Airbyte protocol, but this unblocks us in the immediate while the
         # protocol change is worked on.
         try:
-            return json.loads(log_message.message)
+            return json.loads(log_message.message)  # type: ignore
```

Comment:
json.loads returns `Any`

Reply:
Is this something where the typing is not provided by the lib so we need to use `Any`? Should we we restrict this to `Dict[str, JSONType]` where `JSONType = Union[str, int, float, bool, None, Dict[str, JSONType], List[JSONType]]`?